cmd/orphan-finder/main.go

```diff
@@ -77,7 +83,12 @@ func checkDER(sai certificateStorage, der []byte) (*x509.Certificate, error) {
 	return nil, fmt.Errorf("Existing certificate lookup failed: %s", err)
 }
 
-func parseLogLine(sa certificateStorage, logger blog.Logger, line string) (found bool, added bool) {
+// parseLogLine attempts to parse one log line according to the format used when
+// orphaning certificates. It returns two booleans: The first is true if the
+// line was a match, and the second is true if the certificate was successfully
+// added to the DB. As part of adding a certificate to the DB, it requests a
+// fresh OCSP response from the CA to store alongside the certificate.
+func parseLogLine(sa certificateStorage, ca ocspGenerator, logger blog.Logger, line string) (found bool, added bool) {
```

Comment:
I think in a perfect world this function would be split into two so that one function parses the log lines and another function performs database insertion. Given that this is existing design you're just updating how do you feel about changing `parseLogLine` to be named `storeParsedLogLine` or something else that gives a clearer sense that it's a storage operation as well as a parsing one?

Reply:
Done, thanks!